src/P/List.hs

```diff
@@ -45,3 +46,11 @@ lastMaybe = listToMaybe . reverse
 -- | count the number of elements satisfying a predicate in a list
 count :: (a -> Bool) -> [a] -> Int
 count predicate = length . filter predicate
+
+safeIndex :: [a] -> Int -> Maybe a
+safeIndex xs i
```

Comment:
Wondering if the args should be flipped. (eg. `Int -> [a] -> Maybe a`). That's better for partial application, but not so much infix notation. (I don't have an opinion on it though)